### PR TITLE
Fix loading docker image labels

### LINKED ISSUE #64: Image's ui_name are no longer shown
Docker images with a `ui_name` label no longer has its name shown on the home page of the singleuser app.
It is a retrogression.

### PR DESCRIPTION
**Pull request by kitchoi**
Fix #64

### CODE REVIEW

File: remoteappmanager/docker/image.py [hunk 1]
```diff
@@ -33,7 +33,9 @@ def from_docker_dict(cls, docker_dict):
         except (KeyError, IndexError):
             self.name = ''
 
-        labels = docker_dict.get("Labels")
+        labels = (docker_dict.get("Labels") or
+                  docker_dict.get("Config", {}).get("Labels"))
+
```
Comment (stefanoborini) [inline remoteappmanager/docker/image.py]: If I remember correctly, we decided to use a consistent strategy for this, that is, not to pass the images() dict, but the inspect_image dict instead. We already encountered this problem with the containers IIRC. Please check if we are consistent.

Comment (kitchoi) [inline remoteappmanager/docker/image.py]: > If I remember correctly, we decided to use a consistent strategy for this, that is, not to pass the images() dict, but the inspect_image dict instead.

I am not aware of this decision being made collectively.  Can you point me to the relevant issue please?

**Comment by stefanoborini** [comment]
@kitchoi https://github.com/simphony/simphony-remote/pull/45/files

**Comment by kitchoi** [comment]
@sbo With the current change, `from_docker_dict` can take dictionary from both `images` and `inspect_image`.  I can make it such that it is only compatible with `inspect_image`.  IMHO, I like the dual compatibility (also realise I should add a test for this).  What do you think?

**Comment by stefanoborini** [comment]
@kitchoi I don't have a strong opinion on it, but I do see a problem from the code client side, that you don't know what you are passing. I prefer methods to convey exactly what they operate on, but that's just me. So, your choice, but whatever the choice is, we need to do the same for the container, otherwise the API will differ between the two. Not in this PR, open an issue for it and we'll fix in the slack at the end of the week (if any)

**Comment by kitchoi** [comment]
@sbo I will open an issue for consistency between `docker.container.Container.from_docker_dict` and `docker.image.Image.from_docker_dict`.  I also just added a description in the docstring for the latter.

**Comment by kitchoi** [comment]
@sbo good to merge?

**Pull request merged**